__tests__/topics.test.js

```diff
@@ -327,3 +327,100 @@ describe('CORE: POST /api/articles/:article_id/comments', () => {
       });
   });
 });
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+describe('CORE: DELETE /api/comments/:comment_id', () => {
+  test('204: reponds with no content', () => {
+    return request(app)
+      .delete('/api/comments/1')
+      .expect(204)
+      .then(( {body} ) => {
+        expect(body).toEqual({});
```

Comment:
The body check here is absolutely fine but isn't necceserily required as we can't send back a body with a 204 status code.